autoinstall/FreeBSD/13/installerconfig

```diff
@@ -92,7 +92,13 @@ FreeBSD_latest: {
 EOF
 
 env ASSUME_ALWAYS_YES=YES pkg update -f > /dev/ttyu0
+machtype=$(env | grep MACHTYPE)
```

Comment:
`echo $MACHTYPE` can directly get the value of MACHTYPE or empty if it is undefined. I suggest you adding a line to print the value of machtype for debugging.